bin/video2x_setup.py

```diff
@@ -182,10 +180,10 @@ def download(url, save_path, chunk_size=4096):
     from tqdm import tqdm
     import requests
 
-    output_file = f'{save_path}\\{url.split("/")[-1]}'
-    print(f'Downloading: {url}')
-    print(f'Chunk size: {chunk_size}')
-    print(f'Saving to: {output_file}')
+    output_file = '{}\\{}'.format(save_path, url.split("/")[-1])
```

Comment:
This is a reminder for myself that this should be updated and changed to os.path.join as recommended in #103.